clustered/common/src/main/java/org/ehcache/clustered/common/internal/store/Util.java

```diff
@@ -149,23 +150,27 @@ public ByteBuffer getPayload() {
     };
   }
 
-  public static Object unmarshall(byte[] payload) {
+  public static Object unmarshallWhiteListed(byte[] payload, List<Class> whiteListedClasses) {
```

Comment:
No point in renaming these methods since we're not retaining the originals.

Reply:
done